linera-views/src/backends/rocks_db.rs

```diff
@@ -563,7 +563,7 @@ impl TestKeyValueStore for RocksDbStoreInternal {
 #[derive(Error, Debug)]
 pub enum RocksDbStoreInternalError {
     /// Already existing storage
-    #[error("Already existing storag")]
+    #[error("Already existing storage")]
     StoreAlreadyExist,
```

Comment:
(This should probably be `StoreAlreadyExists` or `StorageAlreadyExists`.)

Reply:
And `"store already exists"` :)